### PR TITLE
Adding more moderators to our communication properties

### PR DESCRIPTION
**Pull request by castrojo**
First cut at expanding our moderator coverage. I wanted to push this early so I can get plenty of review, so there are things missing like an issue template, etc. The general gist:

- Moderator selection process modelled after the k8s membership process. 
- Required yearly checkups for moderator teams.
- A proposed set of open slots for moderators, I just educated-guessed what I think a good number of moderators per property should be, so we can definitely adjust there. 
- A looser set of moderator pro tempore positions to allow for larger coverage for people who can't commit to full time moderation. 

/hold
/sig contributor-experience
/area community-management

### CODE REVIEW

**Comment by castrojo** [comment]
Should probably add a section for moderating teams reporting on their property status yearly to contribex as well.

**Comment by castrojo** [comment]
This is part of fixing this issue: https://github.com/kubernetes/community/issues/3128

**Comment by cblecker** [comment]
Spot review from me looks good. May also worth calling out that the GitHub Administration Team (/github-management#github-administration-team) is responsible for moderation enforcement on GitHub.

File: communication/moderators.md [hunk 1]
```diff
@@ -7,18 +7,29 @@ See our [moderation guidelines](./moderating.md) for policies and recommendation
...
+- Open
 
-### Moderators
+#### Primary Moderators (7)
 
 - Paris Pittman (@parispittman) - PT
 - Jorge Castro (@castrojo) - ET
 - Jaice Singer DuMars - (@jdumars) - PT
 - Louis Taylor (@kragniz)- CET
 - Nikhita Raghunath (@nikhita) - IT
```
Comment (nikhita) [inline communication/moderators.md]: While we are here, can we change my timezone s/IT/IST? There is no timezone called IT. :)

IST = Indian Standard Time

**Comment by castrojo** [comment]
Question for the group, not sure if we should have an escalation policy here, or if we should make a generic one that can cover all things, including moderation.

**Comment by mrbobbytables** [comment]
> Question for the group, not sure if we should have an escalation policy here, or if we should make a generic one that can cover all things, including moderation.

I think we can have a generic escalation policy covering who, when and how things should be escalated. Those sorts of things should be common across the various communication platforms.

**Comment by parispittman** [comment]
should we put the selection of moderators criteria on moderators.md with the list of moderators? that can be the canonical source of who, how whereas moderation.md is the what

File: communication/moderation.md [hunk 2]
```diff
@@ -1,11 +1,75 @@
 # Moderation on Kubernetes Communications Channels
 
 This page describes the rules and best practices for people chosen to moderate
-Kubernetes communications channels. This includes: Slack, the mailing lists
-and _any communication tool_ used in an official manner by the project.
+Kubernetes communications channels. This includes: Slack, the mailing lists and
+_any communication tool_ used in an official manner by the project.
```
Comment (parispittman) [inline communication/moderation.md]: should we say upstream or list them all here? ie - are we responsible for twitter? we need to make the designation here since CNCF owns and/or operates a lot of the properties. we can get the list from the charter.

File: communication/moderators.md [hunk 3]
```diff
@@ -1,28 +1,41 @@
...
 
 ## Mailing Lists
 
 ### kubernetes-dev
 
-### Administrators
+#### Administrators (3)
 
 - Sarah Novotny (@sarahnovotny) - PT
 - Brian Grant (@bgrant0607) - PT
```
Comment (parispittman) [inline communication/moderators.md]: we should ask these folks if they explicitly want to still be included

File: communication/moderation.md [hunk 4]
```diff
@@ -1,11 +1,75 @@
...
 
 - Check the [centralized list of administrators] for contact information.
 
+## Selection of Moderators
+
+Each Kubernetes property has a certain set of [moderators](./moderators.md) who
+are  responsible for keeping it safe and a fun place to participate.
```
Comment (cblecker) [inline communication/moderation.md]: nit extra whitespace

File: communication/moderators.md (hunk 3 above)
Comment (bgrant0607) [reply communication/moderators.md]: I have no interest in being included. Thanks. :-)

File: communication/moderation.md (hunk 2 above)
Comment (castrojo) [reply communication/moderation.md]: I've revised adding a link to the section of our charter which lists them all, also added a bullet saying CNCF owns some of our properties.

File: communication/moderators.md (hunk 3 above)
Comment (sarahnovotny) [reply communication/moderators.md]: i am not able to continue moderating.  thanks for checking.

Comment (idvoretskyi) [reply communication/moderators.md]: I assume, it's about mailing list ownership, not day-to-day operations (moderation).

At the moment @bgrant0607 @sarahnovotny @parispittman and @spiffxp have "owner" permissions at k-dev.

Comment (idvoretskyi) [reply communication/moderators.md]: I assume, it's about mailing list ownership, not day-to-day operations (moderation).

At the moment @bgrant0607 @sarahnovotny @parispittman and @spiffxp have "owner" permissions at k-dev.

File: communication/moderators.md [hunk 5]
```diff
@@ -1,28 +1,44 @@
...
+The following people are responsible for moderating/administrating Kubernetes
+communication channels and their home time zone. See our [moderation
+guidelines](./moderating.md) for policies and recommendations.
 
 ## Mailing Lists
 
 ### kubernetes-dev
 
-### Administrators
+#### Administrators (3)
```
Comment (idvoretskyi) [inline communication/moderators.md]: Commented above.

File: communication/moderation.md [hunk 6]
```diff
@@ -19,8 +85,8 @@ Moderators _MUST_:
 - Take action as specified by these Kubernetes Moderator Guidelines.
   - You are empowered to take _immediate action_ when there is a violation. You
     do not need to wait for review or approval if an egregious violation has
-    occurred. Make a judgement call based on our Code of Conduct and Values
-    (see below).
+    occurred. Make a judgement call based on our Code of Conduct and Values (see
+    below).
   - Removing a bad actor or content from the medium is preferable to letting it
```
Comment (parispittman) [inline communication/moderation.md]: Not preferable - required

File: communication/moderation.md [hunk 7]
```diff
@@ -53,9 +124,8 @@ Moderators _SHOULD_:
 The Kubernetes [Code of Conduct Committee] will have the final authority
 regarding escalated moderation matters. Violations of the Code of Conduct will
 be handled on a case by case basis. Depending on severity this can range up to
-and including removal of the person from the community, though this is
```
Comment (parispittman) [inline communication/moderation.md]: Say something about guidance on removal in escalated situations outside of spammers will come from code of conduct committee.

File: communication/moderators.md [hunk 8]
```diff
@@ -33,14 +49,36 @@ See our [moderation guidelines](./moderating.md) for policies and recommendation
...
+    - Open
+    - Open
+    - Open
+
+## YouTube Channel (9)
+
+### Primary Moderators
 
 - Paris Pittman (@parispittman) - PT
 - Sarah Novotny (@sarahnovotny) - PT
```
Comment (parispittman) [inline communication/moderators.md]: Let’s ask Sarah if she still wants to be here. Also remove bob. Maybe make joe an inactive admin since he doesn’t mod but still keep listed since he is on the account?

File: communication/moderators.md [hunk 9]
```diff
@@ -50,22 +88,48 @@ See our [moderation guidelines](./moderating.md) for policies and recommendation
 - Jorge Castro (@castrojo) - ET
 - Joe Beda - (@joebeda) - PT
 - Jaice Singer DuMars - (@jdumars) - PT
+- Open
```
Comment (parispittman) [inline communication/moderators.md]: Do we have language anywhere about how we should be removing YouTube comments that break code of conduct and that admins trim videos, publish them, set up playlists, etc?

File: communication/moderators.md [hunk 10]
```diff
@@ -50,22 +88,48 @@ See our [moderation guidelines](./moderating.md) for policies and recommendation
...
+
+- Open
+- Open
+- Open
 
 ## Slack
 
+#### Moderators (9)
+
 - Chris Aniszczyk (@caniszczyk) - CT
```
Comment (parispittman) [inline communication/moderators.md]: We should list chris as an owner and an inactive admin

File: communication/moderators.md [hunk 11]
```diff
@@ -50,22 +88,48 @@ See our [moderation guidelines](./moderating.md) for policies and recommendation
...
+- Open
+- Open
+- Open
+- Open
 
 ## Zoom
 
 - Paris Pittman (@parispittman) - PT
 - Jorge Castro (@castrojo) - ET
+- License and main account controlled by the CNCF
```
Comment (parispittman) [inline communication/moderators.md]: We should have at least two other open slots here

File: communication/moderators.md (hunk 9 above)
Comment (parispittman) [reply communication/moderators.md]: Let’s add one more open

File: communication/moderators.md (hunk 8 above)
Comment (castrojo) [reply communication/moderators.md]: Sarah's listed as the primary, I've asked her over mail to transfer, we can then make most of us "communications managers", that will allow uploads and maintenance without too much admin power.

File: communication/moderators.md (hunk 9 above)
Comment (castrojo) [reply communication/moderators.md]: Added an open and added Bob Killen as a pro tempore to reflect his communications manager status in youtube, which wasn't listed in moderators.md. I added 2 bullets in the youtube section, one for the comments, and one for making sure we keep youtube realtime chat off. 

The automated youtube system is similar to discourse, it can detect bad behavior and put it in a queue - I've added an explicit item for the moderators to check that queue. 

The editing and youtube playlist stuff isn't mentioned anywhere, I'll take an item to put that in the youtube.md.

File: .github/MODERATOR_APPLICATION.md [hunk 12]
```diff
@@ -0,0 +1,32 @@
+---
+name: Community Moderator Request
+about: Request moderator priviledges on a Kubernetes property
+title: 'REQUEST: New moderator for <your-GH-handle>' of '<k8s property>'
```
Comment (nikhita) [inline .github/MODERATOR_APPLICATION.md]: The quotes after <your-GH-handle> and before <k8s property> need to be removed for this to work :grimacing:

Comment (nikhita) [reply .github/MODERATOR_APPLICATION.md]: Also..this file needs to go as `.github/ISSUE_TEMPLATE/moderation_application.md` (just tried it on a dummy repo).

File: .github/ISSUE_TEMPLATE/moderator_application.md [hunk 13]
```diff
@@ -0,0 +1,35 @@
+---
+name: Community Moderator Request
```
Comment (cblecker) [inline .github/ISSUE_TEMPLATE/moderator_application.md]: Can we pull this file out into a separate PR? I don't want to hold up the other content, but we don't have issue templates on k/community, and if we add our first one, it will change the workflow for opening new issues completely.

**Comment by parispittman** [comment]
/lgtm

let's do this
